src/SDK/MonitoringApplianceEmulator/Converters/InverseBooleanConverter.cs

```diff
@@ -31,7 +31,7 @@ public object Convert(object value, Type targetType, object parameter, CultureIn
         {
             if (!(value is bool))
             {
-                throw new InvalidOperationException("The value must be boolean");
+                throw new ArgumentException("The value parameter must be boolean");
```

Comment:
>boolean [](start = 73, length = 7)

If you are using ArgumentException, pass the argument name:

new ArgumentException("...", nameof(value))